exporters/otlp/README.md

```diff
@@ -32,7 +32,20 @@ import (
 
 func main() {
```

Comment:
Can we move this example into a [testing example](https://golang.org/pkg/testing/#hdr-Examples)? That way this class of problem is avoided in the future.